src/v/cluster/bootstrap_service.cc

```diff
@@ -10,14 +10,33 @@
 #include "cluster/bootstrap_service.h"
 
 #include "cluster/bootstrap_types.h"
+#include "cluster/cluster_utils.h"
+#include "cluster/cluster_uuid.h"
+#include "cluster/logger.h"
+#include "config/node_config.h"
+#include "features/feature_table.h"
+#include "storage/api.h"
 
 namespace cluster {
 
 ss::future<cluster_bootstrap_info_reply>
 bootstrap_service::cluster_bootstrap_info(
   cluster_bootstrap_info_request&&, rpc::streaming_context&) {
     cluster_bootstrap_info_reply r{};
-    // TODO: add fields!
+    r.broker = make_self_broker(config::node());
+    r.version = features::feature_table::get_latest_logical_version();
+    const std::vector<config::seed_server>& seed_servers
+      = config::node().seed_servers();
+    r.seed_servers.reserve(seed_servers.size());
+    std::transform(
+      seed_servers.cbegin(),
+      seed_servers.cend(),
+      std::back_inserter(r.seed_servers),
+      [](const config::seed_server& seed_server) { return seed_server.addr; });
+    r.empty_seed_starts_cluster = config::node().empty_seed_starts_cluster();
+    r.cluster_uuid = co_await cluster::read_stored_cluster_uuid_front(_storage);
```

Comment:
nit: IMO a cleaner alternative here is to bake `cluster_uuid` into `storage::api` that gets set by `bootstrap_backend`, and have users of the `cluster_uuid` access it through storage. Also it seems odd to hit disk more than once -- why not cache in `storage::api`?